index.js

```diff
@@ -58,6 +59,11 @@ const defaults = userOptions => {
   options.destination = options.destination || options.source;
   if (!options.include || !options.include.length)
     throw new Error("include should be an array");
+  options.include = options.include.map(include =>
+    include.startsWith(options.publicPath)
+      ? include
+      : path.normalize(options.publicPath + include)
```

Comment:
what if options.publicPath does not end with `/`?

Reply:
It will error with a not found or invalid URL. I think that's ok as we expect public path to end with slash.

Reply:
Actually this is fine as include URLs also expect to start with slash.